README.md

```diff
@@ -169,37 +165,37 @@ imports:
 
 ### B. Custom Sets
 
-1. Create `rector.yaml` with desired Rectors:
+1. Create a `rector.yaml` config file with your desired Rectors:
 
     ```yaml
     services:
         Rector\Rector\Architecture\DependencyInjection\AnnotatedPropertyInjectToConstructorInjectionRector:
             $annotation: "inject"
     ```
 
-2. Run on your `/src` directory:
+2. Run Rector on your `/src` directory:
 
     ```bash
     vendor/bin/rector process src --dry-run
     # apply
     vendor/bin/rector process src
     ```
 
-## 3 Steps to Create Own Rector
+## 3 Steps to Create Your Own Rector
 
-First, make sure it's not covered by [any existing Rectors yet](/docs/AllRectorsOverview.md).
+First, make sure it's not covered by [any existing Rectors](/docs/AllRectorsOverview.md).
 
-Let's say we want to **change method calls from `set*` to `change*`**.
+Let's say we want to **change method prefixes from `set*` to `change*`**.
```

Comment:
*method calls* here is technical term from abstract syntax tree nodes, so better keep it